include/revng/Pipeline/Step.h

```diff
@@ -62,6 +75,33 @@ class Step {
   const ContainerSet &containers() const { return Containers; }
   ContainerSet &containers() { return Containers; }
 
+  void setArtifacts(std::string ContainerName, const Kind *ArtifactsKind) {
+    Artifacts = ArtifactsInfo(std::move(ContainerName), ArtifactsKind);
```

Comment:
Can we check if the container with such name exists and return an error (`createStringError`) in case it doesn't?

Reply:
Done